app/views/publishers/_confirm_default_currency_modal.html.slim

```diff
@@ -0,0 +1,22 @@
+javascript:
```

Comment:
We need to ensure that the cards are created and a default currency is chosen even if javascript is off.

Reply:
I will remove all js and make it a static modal.